docs/_docs/refs/mojito-locales.md

```diff
@@ -115,12 +119,15 @@ permalink: /docs/refs/mojito-locales/
 | ms-BN	        | Malay (Brunei Darussalam) |
 | ms-MY	        | Malay (Malaysia) |
 | mt-MT	        | Maltese (Malta) |
+| mus-021	        | Creek (Northern America) |
 | nb-NO	        | Norwegian (Norway) |
 | nl-BE	        | Dutch (Belgium) |
 | nl-NL	        | Dutch (Netherlands) |
 | nn-NO	        | Norwegian (Nynorsk) (Norway) |
 | ns-ZA	        | Northern Sotho (South Africa) |
+| nv-003	        | Navajo (North America) |
```

Comment:
Navajo is a name given by the Spanish. Better to go with Diné

Reply:
YES. This is great insight. To clarify, this downstream tool isn't the place to fix this, but I'd like to support on this.

Term originates here: https://github.com/unicode-cldr/cldr-localenames-full/blob/993632df2f5d6a2d33cbbf40d922474c2482eaca/main/en-001/languages.json#L426

Current CLDR is [v36 (release notes)](http://cldr.unicode.org/index/downloads/cldr-36), and `nv` was added in March 2018 for [v33 (release notes)](http://cldr.unicode.org/index/downloads/cldr-33) (or some time in the year prior to its official version release)

I can try to dig up the mailing lists to see the conversation when this was added. It may be that this conversation happened already. But even so, receptivity to this sort of feedback may have changed.

Reply:
Sorry, requires login to UNICODE ticket tracker: https://unicode-org.atlassian.net/browse/CLDR-13814?jql=text%20~%20%22navajo%22

- `CLDR-13814` [Addition of core data and new locale: Navajo (nv)](https://unicode-org.atlassian.net/browse/CLDR-13814?jql=text%20~%20%22navajo%22)
  - lots of missing information in the registration request -- looks like they could use some support
  - Was submitted by Google on May 24, but they just withdrew on June 8th, as they were not able to "get vetters" in time.

I'm not sure I understand, since it seems it's in there already, but perhaps it's not an official locale yet, as it's lacking some deeper level of detail.

Reply:
I can imagine they might lean on decisions like this:
https://www.indianz.com/News/2017/04/19/navajo-nation-council-rejects-bill-to-ch.asp

but if there are alternative perspectives that have been legitimized through community channels, it could perhaps be raised... maybe there's precedent for some other approach (e.g., multiple entries)

Reply:
Thanks for all the information. Let's talk about this in a chat so I can get the proper perspective on it. What I would like to do is send this out through out social media and gather feedback that way. It would be nice to if someone can point me to something more official.